<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoOptDlg</class>
 <widget class="QWidget" name="VideoOptDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QComboBox" name="SelProf"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Input Device</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="ChkOutFile">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Output File</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame Width x Height, Rate</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="BtnFile">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ChkTimeTag">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Time Tag</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Label4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>File Swap Interva</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SelFileSwap">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Label5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="SelDev">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Label3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>TCP Server Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ChkTcpPort">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Stream</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="EditTcpPort">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>10033</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLineEdit" name="EditFile">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnOk">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Ok</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancle</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Label7">
        <property name="text">
         <string>Frame Caption Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SelCapPos">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Label8">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="EditCapSize">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="SelCapColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Label6">
     <property name="text">
      <string>Codec Quality</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="EditCodecQuality">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>BtnOk</tabstop>
  <tabstop>BtnCancel</tabstop>
  <tabstop>SelDev</tabstop>
  <tabstop>SelProf</tabstop>
  <tabstop>SelCapPos</tabstop>
  <tabstop>EditCapSize</tabstop>
  <tabstop>SelCapColor</tabstop>
  <tabstop>EditCodecQuality</tabstop>
  <tabstop>ChkTcpPort</tabstop>
  <tabstop>EditTcpPort</tabstop>
  <tabstop>ChkOutFile</tabstop>
  <tabstop>BtnFile</tabstop>
  <tabstop>EditFile</tabstop>
  <tabstop>ChkTimeTag</tabstop>
  <tabstop>SelFileSwap</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
